composeApp/src/commonMain/kotlin/org/ooni/probe/ui/choosewebsites/ChooseWebsitesScreen.kt

```diff
@@ -0,0 +1,133 @@
+                        contentDescription = stringResource(Res.string.back),
+                    )
+                }
+            },
+            colors = ColorDefaults.topAppBar(),
+        )
+
+        Box(Modifier.fillMaxSize()) {
+            LazyColumn(
+                contentPadding = WindowInsets.navigationBars.asPaddingValues(),
+            ) {
+                itemsIndexed(state.websites) { index, item ->
+                    OutlinedTextField(
+                        value = item.url,
+                        onValueChange = {
+                            onEvent(
+                                ChooseWebsitesViewModel.Event.UrlChanged(
+                                    index,
+                                    it,
+                                ),
+                            )
+                        },
+                        label = { stringResource(Res.string.Settings_Websites_CustomURL_URL) },
+                        isError = item.hasError,
+                        trailingIcon = {
+                            if (state.canRemoveUrls) {
+                                IconButton(
+                                    onClick = {
+                                        onEvent(ChooseWebsitesViewModel.Event.DeleteWebsiteClicked(index))
+                                    },
+                                ) {
+                                    Icon(
+                                        painterResource(Res.drawable.ic_cancel),
+                                        contentDescription = stringResource(Res.string.Modal_Delete),
+                                    )
+                                }
+                            }
+                        },
+                        modifier = Modifier.fillMaxWidth()
+                            .padding(horizontal = 16.dp)
+                            .padding(top = 16.dp),
+                    )
+                }
+
+                item(key = "Add") {
+                    TextButton(
+                        onClick = { onEvent(ChooseWebsitesViewModel.Event.AddWebsiteClicked) },
+                        modifier = Modifier.fillMaxWidth().padding(16.dp),
+                    ) {
+                        Icon(
+                            painterResource(Res.drawable.ic_add),
+                            contentDescription = null,
+                            modifier = Modifier.padding(end = 8.dp),
+                        )
+                        Text(stringResource(Res.string.Settings_Websites_CustomURL_Add))
+                    }
+                }
+            }
+
+            Button(
+                onClick = { onEvent(ChooseWebsitesViewModel.Event.RunClicked) },
+                modifier = Modifier
+                    .align(Alignment.BottomCenter)
+                    .padding(WindowInsets.navigationBars.asPaddingValues())
+                    .padding(bottom = 16.dp),
+            ) {
```

Comment:
This button eventually blocks the add button.

![Screenshot_20241014_185353](https://github.com/user-attachments/assets/12cb457d-bf99-4072-bae5-5669e0ee4349)